sql/sql_statistics.h

```diff
@@ -326,9 +339,54 @@ class Histogram : public Histogram_base
   /*
     Estimate selectivity of "col=const" using a histogram
   */
-  double point_selectivity(double pos, double avg_sel);
+  double point_selectivity(double pos, double avg_sel) override;
 };
 
+class Histogram_json : public Histogram_base
+{
+private:
+  Histogram_type type;
+  uint8 size; /* Number of elements in the histogram*/
+  uchar *values;
+
+public:
+  bool parse(MEM_ROOT *mem_root, Histogram_type type_arg, const uchar *ptr, uint size) override {return false;}
```

Comment:
Why is this function empty? Don't we have the code already that parses the histogram from its JSON implementation?